### PR TITLE
cfg celery

### PR DESCRIPTION
**Pull request by uover82**

### CODE REVIEW

**Comment by cfregly** [comment]
@uover82 : looks good.  let's discuss and merge when we chat next.

File: scheduler.ml/airflow/config/airflow/airflow.cfg [hunk 1]
```diff
@@ -111,10 +113,12 @@ worker_log_server_port = 8793
 # The Celery broker URL. Celery supports RabbitMQ, Redis and experimentally
 # a sqlalchemy database. Refer to the Celery documentation for more
 # information.
-broker_url = sqla+mysql://airflow:airflow@localhost:3306/airflow
+#broker_url = sqla+mysql://airflow:airflow@localhost:3306/airflow
+broker_url = redis://redis:6379/0
```
Comment (cfregly) [inline scheduler.ml/airflow/config/airflow/airflow.cfg]: @uover82 what is the /0 for at the end of this url?

Comment (cfregly) [inline scheduler.ml/airflow/config/airflow/airflow.cfg]: this should be redis://redis-master:6379/0 per https://github.com/fluxcapacitor/pipeline/blob/b7aeb0f1f3f1aa060d8274a14eb46b3cef640ef7/keyvalue.ml/redis-rc.yaml

File: scheduler.ml/airflow/config/airflow/airflow.cfg [hunk 2]
```diff
@@ -14,12 +14,14 @@ s3_log_folder = None
...
-executor = SequentialExecutor
+#executor = SequentialExecutor
+executor = CeleryExecutor
 
 # The SqlAlchemy connection string to the metadata database.
 # SqlAlchemy supports many different database engine, more information
 # their website
-sql_alchemy_conn = sqlite:////root/airflow/airflow.db
+#sql_alchemy_conn = sqlite:////root/airflow/airflow.db
+sql_alchemy_conn = mysql://root:password@mysql-master/airflow
```
Comment (cfregly) [inline scheduler.ml/airflow/config/airflow/airflow.cfg]: fyi:  i double checked and the `mysql-master` host name is correct per the following kubernetes config

https://github.com/fluxcapacitor/pipeline/blob/b7aeb0f1f3f1aa060d8274a14eb46b3cef640ef7/sql.ml/mysql-rc.yaml

File: scheduler.ml/airflow/config/airflow/airflow.cfg (hunk 1 above)
Comment (uover82) [inline scheduler.ml/airflow/config/airflow/airflow.cfg]: Hi Chris: /0 indicates database number 0, per http://docs.celeryproject.org/en/latest/getting-started/brokers/redis.html. Believe its optional, so no biggie to remove :-)

Comment (uover82) [inline scheduler.ml/airflow/config/airflow/airflow.cfg]: Got this - thanks, Chris. Will do - shall I commit/ merge or merge/ commit/ merge? Or attach updates?

**Comment by cfregly** [comment]
@uover82 did you get a chance to test this out on a kubernetes cluster?  either on [AWS](https://github.com/fluxcapacitor/pipeline/wiki/Setup-Pipeline-AWS) or [Local Laptop](https://github.com/fluxcapacitor/pipeline/wiki/Setup-Pipeline-Mini)?

i want to make sure we didn't break existing functionality like sparkpi and such.

ping me if you want help setting that up.  (it's a 45-60 min task.)

**Comment by uover82** [comment]
No, I haven't. Thanks, yes, let's plan a time to go through this. Local cfg would good to get minikube more integrated too.

**Pull request merged**